src/v2/ion/i18nTransformer.js

```diff
@@ -141,7 +141,7 @@ const I18N_OVERRIDE_MAPPINGS = {
   'enroll-profile.userProfile.firstName': 'oie.user.profile.firstname',
   'enroll-profile.userProfile.email': 'oie.user.profile.primary.email',
 
-  'oie.session.expired': 'oie.idx.session.expired',
+  'idx.session.expired': 'oie.session.expired',
```

Comment:
There is no `oie.session.expired` key coming from API (no such mocks). Neither is there a property `oie.idx.session.expired` in `login.properties`. Updated both.

Reply:
Yep, I removed this line and fixed the key https://github.com/okta/okta-signin-widget/pull/1945/files#diff-391ef791333e16bfeed711da3def133ca2471a6ebbe1e917ecc61330a1530fe3R963